Zend/tests/bug76451.phpt

```diff
@@ -0,0 +1,20 @@
+--TEST--
+Aliases during inheritance type checks affected by opcache
+--INI--
+opcache.enable=1
+opcache.enable_cli=1
+opcache.optimization_level=-1
+--SKIPIF--
+<?php if (!extension_loaded('Zend OPcache') || php_sapi_name() != "cli") die("skip CLI only"); ?>
```

Comment:
The second part of this condition, when it happens? I mean, can we run the tests without been via the CLI?

Reply:
@carusogabriel run-tests can actually execute with php-cgi aswell, see: https://github.com/php/php-src/blob/master/run-tests.php#L123 and below and below :)